<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestResults</class>
 <widget class="QDialog" name="TestResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Grades</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="studyGroupLabel">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>Study group:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="studyGroupComboBox">
           <item>
            <property name="text">
             <string>C112E</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C112A</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>Subject:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="subjectComboBox">
           <item>
            <property name="text">
             <string>Fizica</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Matematica</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Filosofie</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Geografie</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Romana</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="seeGradesButton">
       <property name="text">
        <string>See</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="studentsTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="changeGradeLabel">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>Change grade:</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLineEdit" name="changeGradeLE"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="changeGradeButton">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TestResults</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
